components/prism-editorconfig.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.editorconfig = {
+	// https://editorconfig-specification.readthedocs.io/en/latest/
+	'comment': /[;#].*$/m,
```

Comment:
You don't need the `$`. If the `m` flag is present, `$` will be the same as `(?!.)`. `.*` is greedy, so we know that the next character (if any) will not match `.`, so the `$` == `(?!.)` is redundant.

```suggestion
	'comment': /[;#].*/,
```